File: front-end/project-2/src/components/Item.js
import React from 'react';

export default function Item({name, price, quantity, orderer}) {
    return (
        <div className="item">
            {'Name: ' + name}
            <div className="details">
                <span>{'Price: $' + price}</span>
                <span>{'Quantity: ' + quantity}</span>
                <span>{'Orderer: ' + orderer.name}</span>
            </div>
        </div>
    )
}

File: src/main/java/com/group1/billsplitter/entities/Item.java
package com.group1.billsplitter.entities;

import org.springframework.stereotype.Component;

@Component
public class Item {
    private double price;
    private String name;
    private int quantity;

    public Item(String name, double price, int quantity) {
        this.price = price;
        this.name = name;
        this.quantity = quantity;
    }

    public Item() {
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }
}
